src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssIpConfigCommand.cs

```diff
@@ -107,6 +108,12 @@ public partial class NewAzureRmVmssIpConfigCommand : Microsoft.Azure.Commands.Re
             ValueFromPipelineByPropertyName = true)]
         public string PublicIPPrefix { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        [PSArgumentCompleter("IPv4", "IPv6")]
```

Comment:
It seems that here missed the default value: `IPv4`

Reply:
This is optional.  If a user does not give this parameter, the server sets it as IPv4.